astropy/io/fits/hdu/compressed.py

```diff
@@ -195,7 +195,13 @@ def append(self, card=None, useblanks=True, bottom=False, end=False):
 
         remapped_keyword = self._remap_keyword(card.keyword)
         card = Card(remapped_keyword, card.value, card.comment)
-        self._table_header.append(card=card, useblanks=useblanks,
+
+        # Here we disable the use of blank cards, because the call above to
+        # `Header.append` may have already deleted a blank card in the table
+        # header, thanks to inheritance: `Header.append` calls `del self[-1]`
```

Comment:
It's for comments so it doesn't really matter, but I generally uses this for expressions like `del self[-1]`. Looking at other comments in the file, it uses nothing or sometimes simple quotes, so for consistency I will do the same.